src/views/darkroom.c

```diff
@@ -947,18 +947,15 @@ static void _dev_change_image(dt_develop_t *dev, const int32_t imgid)
   // commit image ops to db
   dt_dev_write_history(dev);
 
+  const int32_t devid = dev->image_storage.id;
```

Comment:
as `devid` is used for OpenCL device I strongly suggest to use `devimgid` or something similar.

Reply:
right and done